### ISSUE TITLE
Won't allow me to push

### ISSUE THREAD

**Issue by Keri-Lynn** [Keri-Lynn/html-document-setup#4]
@thomasjbradley

Hi Thomas,
Today is coding day to get this done.  The spaces aren't going in right and I have to do them manually but as long as I know I don't care at this point.  So now it won't let me push.  I've added the screen shot of the message it is now giving me.  
Thanks,
Keri

![error 150](https://user-images.githubusercontent.com/43151336/47432853-2d868d80-d76d-11e8-83d1-db145ca1ae40.jpg)

**Comment by thomasjbradley** [comment]
Go to the `Repository` menu and choose `Repository Settings`. In the URL does it show your username or my username?

**Comment by Keri-Lynn** [comment]
@thomasjbradley 

Yes, it shows my name.
![screenshot 151](https://user-images.githubusercontent.com/43151336/47439589-a3452600-d77a-11e8-9e57-0df820e9fb4a.png)

**Comment by thomasjbradley** [comment]
I meant in the GitHub Desktop application.

**Comment by Keri-Lynn** [comment]
@thomasjbradley
![error 150](https://user-images.githubusercontent.com/43151336/47450339-fc20b880-d792-11e8-88c4-5c8e1537d3b4.jpg)

Nope, nobody's name.

**Comment by thomasjbradley** [comment]
In the GitHub Desktop app, in the menu, go to `Repository > Repository Settings`.
There should be a URL on the little screen that pops up. Does it have your username in it?

**Comment by Keri-Lynn** [comment]
@thomasjbradley

Nope, nobody's name. :(

![screenshot 153](https://user-images.githubusercontent.com/43151336/47517312-cb558780-d855-11e8-950e-9f777363bc87.png)

**Comment by thomasjbradley** [comment]
Yep! That `acgd-webdev-1` part of the URL should be changed to your GitHub username.
